tests/foreman/ui/test_host.py

```diff
@@ -2754,34 +2754,53 @@ def test_positive_set_multi_line_and_with_spaces_parameter_value(
 class TestHostAnsible:
     """Tests for Ansible portion of Hosts"""
 
-    @pytest.mark.stubbed
-    @pytest.mark.tier2
-    def test_positive_host_role_information(self):
-        """Assign Ansible Role to a Host and an attached Host group and verify that the information
-        in the new UI is displayed correctly
 
-        :id: 7da913ef-3b43-4bfa-9a45-d895431c8b56
+@pytest.mark.tier2
+def test_positive_host_role_information(target_sat):
+    """Assign Ansible Role to a Host and verify that the information
+    in the new UI is displayed correctly
 
-        :caseComponent: Ansible
+    :id: 7da913ef-3b43-4bfa-9a45-d895431c8b56
 
-        :Team: Rocket
+    :caseComponent: Ansible
 
-        :CaseLevel: System
+    :Team: Rocket
 
-        :Steps:
-            1. Register a RHEL host to Satellite.
-            2. Import all roles available by default.
-            3. Create a host group and assign one of the Ansible roles to the host group.
-            4. Assign the host to the host group.
-            5. Assign one role to the RHEL host.
-            6. Navigate to the new UI for the given Host.
-            7. Select the 'Ansible' tab, then the 'Inventory' sub-tab.
+    :CaseLevel: System
 
-        :expectedresults: Roles assigned directly to the Host are visible on the subtab, and
-            roles assigned to the Host Group are visible by clicking the "view all assigned
-            roles" link
+    :Steps:
+        1. Register a RHEL host to Satellite.
+        2. Import all roles available by default.
+        3. Assign one role to the RHEL host.
+        4. Navigate to the new UI for the given Host.
+        5. Select the 'Ansible' tab, then the 'Inventory' sub-tab.
 
-        """
+    :expectedresults: Roles assigned directly to the Host are visible on the subtab.
+
+    """
+    SELECTED_ROLE = 'RedHatInsights.insights-client'
+
+    host = target_sat.api.Host().create()
+    location = (
+        target_sat.api.Location().search(query={'search': f'id = {host.location.id}'})[0].name
+    )
+    organization = (
+        target_sat.api.Organization()
+        .search(query={'search': f'id = {host.organization.id}'})[0]
+        .name
+    )
+    id = target_sat.api.SmartProxy(name=target_sat.hostname).search()[0].id
```

Comment:
If you're just trying to get the capsule you can use
```suggestion
    id = target_sat.nailgun_smart_proxy
```
I would also suggest changing id to something more descriptive like `capsule_id` or `proxy_id`.